src/services/inplayer.account.service.ts

```diff
@@ -328,3 +333,75 @@ function parseJson(value: string, fallback = {}) {
 export const canUpdateEmail = false;
 
 export const canChangePasswordWithOldPassword = true;
+
+export const initCustomerExtras = async (): Promise<ExternalData> => {
```

Comment:
I think it would be advisable to put watch list stuff into its own service. We already have favorites and continue watching controllers, and this would avoid coupling a lot of things like this to the account and user.

Reply:
yes, this makes sense and I already fixed that as you said